docs/src/architecture/accounts.md

```diff
@@ -13,63 +16,56 @@ In the above:
 * **Vault** is collection of assets stored in an account.
 * **Code** is a collection of functions which define an external interface for an account.
 
-## Account ID
-8 bytes (60 bit) long identifier for the account. The first three significant bits specify its type and the storage mode. There are four types of accounts in Miden that can be stored in two differnt ways:
-
-### Regular account with updatable code
-This account type will be used by most users. They can specify and change their code and use this account for a wallet. The Account ID will start with `00`.
-
-### Regular account with immutable code
-This account type will be used by most regular smart contracts. Once deployed the code should not change and no one should be able to change it. The Account ID will start with `01`.
-
-### Fungible asset faucet with immutable code
-Assets need to be created by accounts in Miden. The Account ID will start with `10`.
-
-### Non-fungible asset faucet with immutable code
-Assets need to be created by accounts in Miden. The Account ID will start with `11`.
-
-### Account storage mode
-Account data - stored by the Miden Node - can be public or private. The third most significant bit of the ID specifies whether the account data is public `0` or private `1`.
+### Account ID
+8 bytes (60 bit) long identifier for the account. The first three significant bits specify its type and the [storage mode](https://0xpolygonmiden.github.io/miden-base/architecture/accounts.html#account-storage-modes). There are four types of accounts in Miden:
```

Comment:
8 bytes != 60 bits.  Currently we use 1 field element (~63 bits) to represent the account id.  We do have plans to change this but the documentation is leading the code at the minute.  I'm not sure if we should do this?

Reply:
You are right, we should not. Let's use the 63 bits